HIVE-29131:  [Website] Update the hive website and the docs theme

simhadri-g opened the pull request:
Hi Everyone, 
I have added a new docs theme to the hive website and updated the homepage . 

I have hosted the new changes here, please take a look: https://simhadri-g.github.io/hive-site/ 

Feedback is most welcomed!
Thanks!

deniskuzZ commented on themes/hive/layouts/partials/repl.html:
```diff
@@ -17,7 +17,7 @@
-    <div id="repl" data-termynal data-ty-typeDelay="40" data-ty-lineDelay="70" style="width: 830px;">
+    <div id="repl" data-termynal data-ty-typeDelay="40" data-ty-lineDelay="70" style="width: 830px; font-size: 0.9em;" >
         <span data-ty="input">jdbc:hive2://> repl dump src with (</span>
         <span data-ty="input">. . .>   'hive.repl.dump.version'= '2',</span>
```
could you replace `. . .>` to `->` in 4 places

simhadri-g replied:
Sure, i just pushed a fix.
Thanks!

deniskuzZ commented:
Looks amazing!
 
Do you think it would be better if we put the `release-banner `in the top right corner?

simhadri-g commented:
> Do you think it would be better if we put the release-banner in the top right corner or just at the top?

I am fine with  both the options, this was just initial implementation. We can progressively like improve this :)

zhangbutao commented:
<img width="1848" height="1388" alt="image" src="https://github.com/user-attachments/assets/f3d368b7-080a-481f-aa82-0ab2db310407" />
The topic navigation on the left can be unrelated to the content displayed on the right. I find this quite confusing. In my opinion, under normal circumstances, the content on the right should belong to a part of the topic navigation on the left.

deniskuzZ commented:
i can still see bad text alignment
<img width="552" height="420" alt="Screenshot 2025-08-11 at 21 22 54" src="https://github.com/user-attachments/assets/dd79ad00-5409-400c-b05f-2d332cc49b22" />

simhadri-g commented:
yes it hosted on the same link, i just rebased. Try reloading the page. 

Also the text alignment changes with screen size.  Not sure how to make it uniform.

<img width="463" height="353" alt="image" src="https://github.com/user-attachments/assets/1fcdf61f-3ebf-4f28-96bc-c0d1f2621671" />

deniskuzZ commented:
+ In my opinion "[Apache Hive : Introduction to Apache Hive](https://simhadri-g.github.io/hive-site/docs/latest/introduction-to-apache-hive_283118337/)" should be the first

simhadri-g commented:
Oh i see the table, will fix that. 

 I was talking about after clicking the java doc link for 4.0, i see nothing in the svn link

<img width="733" height="520" alt="image" src="https://github.com/user-attachments/assets/dedf05f9-1e97-4ae2-9042-f0c0d0413477" />

deniskuzZ commented:
https://simhadri-g.github.io/development/desingdocs/ -> 404

deniskuzZ commented:
> I was talking about after clicking the java doc link for 4.0, i see nothing in the svn link

it was loading few days ago for me, maybe some infra stuff

deniskuzZ commented:
btw, I've already approved, so maybe we can merge this PR and fix new findings in a follow-up?

ayushtkn commented:
I don't think adding the logo is a problem, if someone else wants to add their logo we are happy to add their company logo as well. Even Kafka has it:
https://kafka.apache.org/powered-by

simhadri-g commented:
Thanks , @deniskuzZ , @ayushtkn and @zhangbutao . 

I have addressed the review comments, please take a look: https://simhadri-g.github.io/hive-site/ 

Regarding this, https://github.com/apache/hive-site/pull/59#issuecomment-3190370248 , We are see this because of recent changes made in the markdown file. The new website just render the content. We will need to update the mark down files and remove that. I think we should do that in a follow up PR.

ayushtkn commented on config.toml:
```diff
@@ -1,4 +1,4 @@
-baseURL = 'https://hive.apache.org'
+baseURL = 'https://simhadri-g.github.io/hive-site/'
```
Do revert this back before merging

ayushtkn reviewed (approved):
LGTM. The new website looks good and appealing to me. Lets merge it and if there are issues lets address them as new PRs
